server/index.js

```diff
@@ -48,14 +48,32 @@ async function setupApp() {
 
   });
 
+  app.post('/users', async (req, res) => {
+    try {
+      const { name, email, password, birthdate, notification_preference, biography, terms_agreed, newsletter_subscribed } = req.body;
+      const user = await pgClient.query(`SELECT * FROM  messages WHERE email = $1`, [email]);
```

Comment:
Backtick is when we need to interpolate. We should not interpolate any SQL statement otherwise we'll introduce SQL injection. (You didnt sub anything here, but just in case)

```suggestion
      const user = await pgClient.query('SELECT * FROM  messages WHERE email = $1', [email]);
```